packages/admin-ui/src/radio/radio-group.tsx

```diff
@@ -28,8 +28,8 @@ export const RadioGroup = createComponent<'fieldset', RadioGroupOptions>(
     return useElement('fieldset', {
       ...restProps,
       children: (
-        <FormControl error={error} optional={optional}>
-          <FormControlLabel as="legend">{label}</FormControlLabel>
+        <FormControl>
+          <FormControlLabel optional={optional}>{label}</FormControlLabel>
```

Comment:
No need to check for `label`'s existence here?

Reply:
This label is mandatory, so no.

Reply:
Got it